### DISCUSSION TITLE
syntax highlight not working in vscode?

### DISCUSSION THREAD

**Discussion by Zzzen** [zigtools/zls#594] [Q&A] [answered]
I have installed zls extension and logs seem to be OK. 

![image](https://user-images.githubusercontent.com/6630042/185771978-22f3b88c-432a-4b86-aa19-c0731abbd9e4.png)

**Comment by nektro** [comment, marked as answer]
need both the `tiehuis.zig` and `AugusteRame.zls-vscode` extensions for it to work properly

**Reply by Zzzen** [reply]
Thanks. IntelliSense works now but code completion does not. I would expect a list of suggestions popping up when `std.` is typed.

https://user-images.githubusercontent.com/6630042/185773246-33899c9d-e378-47d2-8649-81ed3dcc3aaa.mp4

**Reply by nektro** [reply]
the options are in the extension settings of `AugusteRame.zls-vscode`

**Reply by Zzzen** [reply]
Which options? I've tried toggling `Enable_import_embedfile_argument_completions`/`Operator_completions`, nothing changes

**Reply by nektro** [reply]
oh maybe not all of the options are transferred yet, try running `./zls config`

**Comment by truemedian** [comment]
If you want to use the semantic highlighting that zls provides, you need to enable semantic highlighting in vscode `editor.semanticHighlighting.enabled`. No other extensions are required
